server/settings_test.go

```diff
@@ -51,6 +52,53 @@ func TestSettings(t *testing.T) {
 
 			defer restoreSettingsDefaults(t)
 
+			t.Run("Updates", func(t *testing.T) {
+				t.Run("DisableAndEnableUpdatesSettingsUpdate", func(t *testing.T) {
+					defer restoreSettingsDefaults(t)
+					res, err := serverClient.Default.Server.ChangeSettings(&server.ChangeSettingsParams{
+						Body: server.ChangeSettingsBody{
+							EnableUpdates: true,
+						},
+						Context: pmmapitests.Context,
+					})
+					require.NoError(t, err)
+					assert.False(t, res.Payload.Settings.UpdatesDisabled)
+					assert.Empty(t, err)
+
+					resg, err := serverClient.Default.Server.GetSettings(nil)
+					require.NoError(t, err)
+					assert.False(t, resg.Payload.Settings.UpdatesDisabled)
+
+					res, err = serverClient.Default.Server.ChangeSettings(&server.ChangeSettingsParams{
+						Body: server.ChangeSettingsBody{
+							DisableUpdates: false,
```

Comment:
it does nothing actually.
Can we check that `DisableUpdates: true` disables updates?

Reply:
Fixed.